context-js/tests/parse_compressed_bundle.test.ts

```diff
@@ -103,6 +103,14 @@ describe("context-js", () => {
         command_line: "trunk-analytics-cli upload --token=***",
       },
     ],
+    [
+      "V0_6_3",
+      {
+        num_tests: faker.number.int(100),
+        num_files: faker.number.int(100),
+        command_line: "trunk-analytics-cli upload --token=***",
```

Comment:
I think this is missing `id_v2`? I'm surprised this compiled (`pnpm i --dir context-js; pnpm --dir context-js run build_and_test`)